plugin/someteam.example.com/v1/bashedconfigmap/BashedConfigMap_test.go

```diff
@@ -30,7 +30,4 @@ kind: ConfigMap
 metadata:
   name: example-configmap-test
 `)
-	if m.Resources()[0].NeedHashSuffix() != true {
-		t.Errorf("expected resource to need hashing")
-	}
```

Comment:
This check has to be removed because the last step of running the generator is to remove the build annotations, after which we no longer have access to the options which were stored in the annotations.

Reply:
Is that problem with the test harness, or does it actually mean that generators (at least the Go type) used to be able to request hashing directly via Resource and that will stop working?

Reply:
It's a problem with the test harness but it's actually not hard to fix. I'll update it.

Reply:
Fixed